cabal.project

```diff
@@ -47,7 +47,14 @@ index-state: 2022-03-08T10:53:01Z
 constraints:
     hyphenation +embed
 
+source-repository-package
```

Comment:
```suggestion
-- This is benign and won't affect our ability to release to Hackage, 
-- because we only depend on `ekg-wai` when a non-default flag
-- is turned on.
source-repository-package
```